llvm/test/CodeGen/SPIRV/extensions/SPV_KHR_bit_instructions/cl_khr_extended_bit_ops.cl

```diff
@@ -0,0 +1,1191 @@
+// RUN: %clang_cc1 -triple spir-unknown-unknown -O1 -cl-std=CL2.0 -fdeclare-opencl-builtins -finclude-default-header -emit-llvm-bc %s -o %t.bc
```

Comment:
I am afraid adding backend tests based on Clang may not be feasible. Changes to Clang might affect the LLVM IR output of these tests and I would not expect anyone contributing to Clang to check against tests in SPIR-V backend. The same goes the other way around. Also, some build bots running backend tests do not build Clang (e.g. our SPIR-V Tests GitHub Action).

Would you be able to rewrite the test in LLVM IR? It still might be a good idea to include a part of .cl source in a comment to help future contributors understand the test case.

Reply:
Done in https://github.com/llvm/llvm-project/pull/120571/commits/c3fe99965b0cb9159efc18d50bc33e0173508ecb.